helm/oauth2-proxy/templates/secret-alpha.yaml

```diff
@@ -7,8 +7,8 @@
 apiVersion: v1
 kind: Secret
 metadata:
-{{- if .Values.alphaConfig.annotations }}
-  annotations: {{- toYaml .Values.alphaConfig.annotations | nindent 4 }}
+{{- with .Values.alphaConfig.annotations }}
+{{ tpl ( toYaml . ) $ | indent 4 }}
```

Comment:
Can you put it back?
```
annotations:
```

Reply:
Sorry, copy-pasted to much - it's back, thanks for noticing!